src/explore-education-statistics-admin/src/components/editable/__tests__/ReorderableAccordion.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import noop from 'lodash/noop';
+import Button from '@common/components/Button';
+import ReorderableAccordion from '../ReorderableAccordion';
+
+describe('ReorderableAccordion', () => {
+  test("it doesn't show controls if `canUpdateRelease` is false", () => {
+    const canUpdateRelease = false;
+
+    render(
+      <ReorderableAccordion
+        canUpdateRelease={canUpdateRelease}
+        id="test"
+        onReorder={() => noop}
+        heading="test"
+        onSectionOpen={() => noop}
+        onToggleAll={() => noop}
+      >
+        {canUpdateRelease && <Button>Reorder</Button>}
+      </ReorderableAccordion>,
+    );
+
+    expect(
+      screen.queryByRole('button', { name: 'Reorder' }),
+    ).not.toBeInTheDocument();
+  });
+
+  test('it shows controls if `canUpdateRelease` is true', () => {
+    const canUpdateRelease = true;
+    render(
+      <ReorderableAccordion
+        canUpdateRelease={canUpdateRelease}
+        id="test"
+        onReorder={() => noop}
+        heading="test"
+        onSectionOpen={() => noop}
+        onToggleAll={() => noop}
+      >
+        {canUpdateRelease && <Button>Reorder</Button>}
```

Comment:
Same as earlier - I'm not sure why we need this?

The assertion should be able to look for a single 'Reorder sections' (as suggested) button.